test/stresstest/config.py

```diff
@@ -0,0 +1,13 @@
+import configparser
+import os
+
+def getConfig():
+
+    config = configparser.ConfigParser()
+    home = os.path.expanduser("~")
+    config_path = home + '/kuksa.val/test/stresstest/config.ini'
```

Comment:
I don't have that path on my machine.
I'd say it's better if you deduce that path from `config.py`'s path which you can get through `__file__`. Then you could do:
```python
config_path = pathlib.Path(__file__).resolve().parent
```
`resolve()` will resolve any symbolic links and turn your path into an absolute one.